src/test/kotlin/no/nav/familie/pdf/pdf/PdfValidatorTest.kt

```diff
@@ -29,6 +29,7 @@ class PdfValidatorTest {
         // Act
         val result = PdfValidator.validerPdf(pdfBytes, standard)
         // Assert
-        assertTrue(result.samsvarer, "Pdf-en samsvarer ikke med standarden $standard med feilen ${result.feiletRegel}")
+        // Spesial-tilfelle fordi regel 8.8-2 ikke er oppfylt ved klikkbar lenke i innholdsfortegnelsen
```

Comment:
Brude ikke dette heller være en egen test hvor au2 feiler og at testen da blir grønn? For hvis det skjer noe i fremtiden med lenker i pdf-er så er det fint å få beskjed om det har ordnet seg. Da kan også metainnhodlet oppdateres til å samsvare med au2

Reply:
Hmm 🤔 Forskjellen blir da at alle UA2 feil kommer til å bli "grønn". Her vil kun den regelen for klikkbar lenke i innholdsfortegnelsen feile og bli "grønn". Så fort en annen regel vil feile her, vil den bli plukket opp av testen, noe den testen der ua2 feiler ikke kommer til å gjøre.
Eller misforstår jeg deg?